components/applications-service/pkg/storage/postgres/service_groups.go

```diff
@@ -24,47 +24,8 @@ const (
 	// 1) Counts health of services (How many ok, critical, warning and unknown) and its total
 	// 2) Calculates the percentage of services with an ok health.
```

Comment:
we might need to update all the comments so that developers don't get confused with them.